### ISSUE TITLE
部分机型UICollectionView，点击无反应，而且并不是必现，每次重启手机第一次打开App机率高点

### ISSUE THREAD

**Issue by CaptainErich** [tigerAndBull/TABAnimated#231]
### New Issue Checklist

* [ ] I have read the [Documentation](https://github.com/tigerAndBull/TABAnimated/tree/master/Documents)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/TABAnimated/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. iOS / macOS
 Platform Version        | e.g. 12.0 / 10.14.0
 TABAnimated Version      | e.g. 5.0.0 / 4.4.0
 Xcode Version           | e.g. Xcode 11 / Xcode 10
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by tigerAndBull** [comment]
有图层显示吗，骨架屏不会修改collectionView的响应

**Comment by kaelinda** [comment]
我也遇到了，是end状态没更新过来导致的。
手动把state = .end可以临时解决

**Comment by tigerAndBull** [comment]
> 我也遇到了，是end状态没更新过来导致的。 tab_endAnimation()之后，手动把state = .end可以临时解决

能提供下具体复现场景吗

**Comment by tongxiaobo** [comment]
tableview也有类似的情况
